src/cobald/daemon/runners/meta_runner.py

```diff
@@ -60,6 +60,8 @@ def run(self):
                 asyncio_main_run(root_runner=thread_runner)
             else:
                 thread_runner.run()
+        except KeyboardInterrupt:
+            self._logger.info("runner interrupted")
```

Comment:
Thanks for adding this. Doesn't look right to exit COBalD always with a stack trace.

Reply:
Indeed, this was annoying me for ages. 😄